backend/src/procedure/docker.rs

```diff
@@ -479,27 +475,28 @@ async fn buf_reader_to_lines(
     Ok(output)
 }
 
-async fn max_buffer(
+async fn max_by_lines(
```

Comment:
So make it so I don't have to go to vec to early and I avoided the issue that the buffer was wrong for the serialize, so glad that got fixed. And this makes more sense to keep it in string